tests/x11/gnome_console.pm

```diff
@@ -0,0 +1,30 @@
+# SUSE's openQA tests
+#
+# Copyright 2025 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: gnome-console
+# Summary: Basic functionality of gnome console
+# - Launch "gnome-console" and confirm it is running
+# - Open a second tab
+# - Type "If you can see this text gnome-console is working."
+# - Close gnome-terminal
+# Maintainer: Santiago Zarate <santiago.zarate@suse.com>
+
+use base "x11test";
+use strict;
+use warnings;
+use testapi;
+
+sub run {
+    my ($self) = @_;
+    mouse_hide(1);
+    x11_start_program('kgx');
+    send_key "ctrl-shift-t";
+    assert_screen "gnome-console-second-tab";
+    $self->enter_test_text('gnome-terminal', cmd => 1);
```

Comment:
intentionally entering gnome-terminal?

Reply:
Unintentional, fixed now, and needles updated properly